packaging/README.release.md

```diff
@@ -24,7 +24,7 @@ an entry like the following:
 
 ## Tagging
```

Comment:
Please also add a checkbox for the changelog creation.

Reply:
Added, will push in the next commit.